components/flow-orchestration-framework/org.wso2.carbon.identity.flow.execution.engine/src/main/java/org/wso2/carbon/identity/flow/execution/engine/FlowExecutionService.java

```diff
@@ -90,12 +94,22 @@ public FlowExecutionStep executeFlow(String tenantDomain, String applicationId,
                 }
             }
             if (STATUS_COMPLETE.equals(step.getFlowStatus())) {
```

Comment:
Is this registration flow or any flow ?

Reply:
Fixed 8d2f7ed73ccf54d1cbc7c85e4a28a8b696e2169f